optimade/server/middleware.py

```diff
@@ -297,6 +297,14 @@ class AddWarnings(BaseHTTPMiddleware):
     the response's body content is actually streamable, by breaking it down into
     chunks of the original response's chunk size.
 
+    !!! warning "Important"
+        It is **recommended** to add this middleware as the _last one_ to your application.
```

Comment:
```suggestion
        It is **recommended** to add this middleware as the _last one_ to your application.
        This can be achieved by applying `AddWarnings` _after_ all other middleware
        with the `.add_middleware()` method, or by initialising the app with a middleware list 
        in which `AddWarnings` appears _first_.
```